server/db/fastfood.sql

```diff
@@ -3,20 +3,20 @@ CREATE TABLE IF NOT EXISTS users (
    firstname  VARCHAR(255) NOT NULL,
    lastname  VARCHAR(255) NOT NULL,
    email  VARCHAR(255) NOT NULL UNIQUE,
-   telephone  VARCHAR(255) NOT NULL,
+   telephone  VARCHAR(255) NOT NULL UNIQUE,
```

Comment:
This field should be nullable because at first, you may not need  to get the user phone number